account_analytic_default_account/views/account_analytic_default_account_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Camptocamp SA
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+        <record id="view_account_analytic_default_account_tree" model="ir.ui.view">
```

Comment:
optional:
use original id (view_account_analytic_default_tree)

Reply:
@tbaden  thanks for review, updated